src/main/java/mate/academy/springboot/datajpa/controllers/CategoryController.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.springboot.datajpa.controllers;
+
+import mate.academy.springboot.datajpa.dto.mapper.CategoryMapper;
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.models.Category;
+import mate.academy.springboot.datajpa.services.CategoryService;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/categories")
+public class CategoryController {
+    private final CategoryService categoryService;
+    private final CategoryMapper mapper;
+
+    public CategoryController(CategoryService categoryService, CategoryMapper mapper) {
+        this.categoryService = categoryService;
+        this.mapper = mapper;
+    }
+
+    @GetMapping("/inject")

```

Comment:
The '/inject' GET mapping should be removed or replaced with a proper method for creating multiple categories at once, as it is not a good practice to use fixed data in the controller.

Reply:
It is better to create an Injector class and in the method indicated by the @PostConstruct, call all the data that needs to be recorded in the database

Reply:
Ok, I forgot to delete these data, it were necessary for cheking workability of application).